media-api/app/lib/elasticsearch/ElasticSearch.scala

```diff
@@ -128,6 +129,43 @@ object ElasticSearch extends ElasticSearchClient with SearchFilters with ImageFi
       }
   }
 
+  def siblingLabelsSearch(siblingLabel: String)(implicit ex: ExecutionContext): Future[AggregateSearchResults] = {
+    val name = "siblingLabels"
+    val lastModifiedField = "lastModified"
+    val labelsField = editsField("labels")
+
+    val sortByDateAggr =
```

Comment:
As much as ordering by `lastModified` puts something at the top of the list when you add a label, another record would usurp it if you edit, say, the metadata. We might want to add the idea of lastModified to each `Resource`? Perhaps something for the argo-rest/spec?